playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java

```diff
@@ -250,7 +261,26 @@ public void showVideoSubtitles() {
     public SubtitleView getSubtitleView() {
         return subtitleView;
     }
-    
+
+    @Override
+    public void setArtworkViewVisibility(boolean isVisible) {
+        if (artworkDrawable != null) {
+            artworkView.setVisibility(isVisible ? VISIBLE : GONE);
+        } else {
+            artworkView.setVisibility(GONE);
+        }
+
+        if (shutterView != null && shutterView.getVisibility() == VISIBLE) {
```

Comment:
1. should be called only if isVisible is true
2.should be moved to be the first thing done in the method

Reply:
This logic is required for shutterview to keep it hide if the artworkview is there. Other wise it is visible in bg/fg case.